packages/codegen/src/generator/utils.ts

```diff
@@ -7,7 +11,19 @@ export const commentBlock = (docs: string | string[]) => {
     return `
 /**
 ${docs.map((line) => `* ${line.replaceAll(/\s+/g, ' ').trim()}`).join('\n')}
- **/
-      `;
+**/
+        `;
   }
-}
+};
+
+export const resolveFilePath = (relativePath: string | string[]) => {
+  relativePath = Array.isArray(relativePath) ? relativePath : [relativePath];
+
+  return path.resolve(process.cwd(), ...relativePath);
+};
+
+export const PRETTIER_FORMAT_OPTION = await prettier.resolveConfig(resolveFilePath('./.prettierrc.js'));
+
+export const format = (tsInput: string) => {
```

Comment:
We should rename this to make the intention more clear & add a return type for this as well.
```suggestion
export const beautifySourceCode = (source: string): Promise<string> => {
```